Make.inc

```diff
@@ -888,6 +893,7 @@ endif
 ifeq ($(OS), FreeBSD)
 JLDFLAGS := -Wl,-Bdynamic
 OSLIBS += -lelf -lkvm -lrt -Wl,--export-dynamic -Wl,--version-script=$(JULIAHOME)/src/julia.expmap $(NO_WHOLE_ARCHIVE) $(LIBUNWIND)
+USE_SYSTEM_CURL := 1
```

Comment:
i thought i was having trouble building on of the dependencies (one of the test files was missing one of its headers), but it's working now, so not sure.